sql/hive/src/test/scala/org/apache/spark/sql/hive/StatisticsSuite.scala

```diff
@@ -144,6 +144,7 @@ class StatisticsSuite extends QueryTest with BeforeAndAfterAll {
         expectedAnswer: Seq[Row],
         ct: ClassTag[_]) = {
       before()
+      conf.setConf("spark.sql.autoSortMergeJoin", "false")
```

Comment:
Why do we need to make sure it is off? The planner should prefer broadcast join, right?